From c7f5ded26990b71164e143abbbf37c737de7d7a4 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 16 Mar 2023 13:57:03 +1100
Subject: [PATCH] always link in compiler-rt on windows

---
 flang/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flang/CMakeLists.txt b/flang/CMakeLists.txt
index b048ea220e..99887069b3 100644
--- a/flang/CMakeLists.txt
+++ b/flang/CMakeLists.txt
@@ -46,7 +46,7 @@ endif()
 include(GNUInstallDirs)
 
 # MSVC + clang-cl build requires clang_rt.builtin.${target} library
-if (MSVC AND CMAKE_CXX_COMPILER_ID MATCHES Clang)
+if (MSVC)
   include(HandleCompilerRT)
   find_compiler_rt_library(builtins CLANG_RT_BUILTINS_LIBRARY)
   get_filename_component(LIBDIR "${CLANG_RT_BUILTINS_LIBRARY}" DIRECTORY)
-- 
2.38.1.windows.1

