From e2fd192e8e616c73162e9c7842d23321ede09358 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 10 Jan 2025 13:23:00 +0100
Subject: [PATCH 1/2] DEBUG: flang_rt paths

---
 flang-rt/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/flang-rt/CMakeLists.txt b/flang-rt/CMakeLists.txt
index b3b6e00f7c0c..375b5976bab1 100644
--- a/flang-rt/CMakeLists.txt
+++ b/flang-rt/CMakeLists.txt
@@ -115,6 +115,12 @@ endif ()
 set(FLANG_RT_INSTALL_RESOURCE_PATH "${FLANG_RT_INSTALL_RESOURCE_PATH_DEFAULT}"
     CACHE PATH "Path to install runtime libraries to (default: clang resource dir)")
 extend_path(FLANG_RT_INSTALL_RESOURCE_LIB_PATH "${FLANG_RT_INSTALL_RESOURCE_PATH}" "${toolchain_lib_subdir}")
+message("LLVM_TREE_AVAILABLE: ${LLVM_TREE_AVAILABLE}")
+message("toolchain_lib_subdir: ${toolchain_lib_subdir}")
+message("FLANG_RT_OUTPUT_RESOURCE_DIR: ${FLANG_RT_OUTPUT_RESOURCE_DIR}")
+message("FLANG_RT_INSTALL_RESOURCE_PATH: ${FLANG_RT_INSTALL_RESOURCE_PATH}")
+message("FLANG_RT_OUTPUT_RESOURCE_LIB_DIR: ${FLANG_RT_OUTPUT_RESOURCE_LIB_DIR}")
+message("FLANG_RT_INSTALL_RESOURCE_LIB_PATH: ${FLANG_RT_INSTALL_RESOURCE_LIB_PATH}")
 cmake_path(NORMAL_PATH FLANG_RT_OUTPUT_RESOURCE_DIR)
 cmake_path(NORMAL_PATH FLANG_RT_INSTALL_RESOURCE_PATH)
 # FIXME: For the libflang_rt.so, the toolchain resource lib dir is not a good
